scanner.py

```diff
@@ -19,10 +20,22 @@
 # Check what time the scan started
 t1 = datetime.now()
 
+# Getting port range values from config.json
+try:
+    with open('config.json') as config_file:
+        config = json.load(config_file)
+    range_high = int(config['range']['high'])
+    range_low = int(config['range']['low'])
+    
+except FileNotFoundError:
+    print("config.json file not found")
+except ValueError:
+    print("Kindly check the json file for appropriateness of range")
+
 # scanning the port only in range of (1, 8888)
 
 try:
-    for port in range(1,8888):  
+    for port in range(range_low,range_high):  
```

Comment:
@ramesh-jr Linting issue here.

Reply:
@vinitshahdeo Hi,Vinit. I couldn't see any linting issue on this line. Could you kindly elaborate?